card.io/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,9 @@
 
     <uses-permission android:name="android.permission.CAMERA"/>
     <uses-permission android:name="android.permission.VIBRATE"/>
+    <!-- We need this for tessdata-->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
There should be no need to write it to external storage, the internal private storage for the app should be fine. I've had to do something similar with an OpenVPN binary that needs to read a configuration file from disk, it does need to be written to storage instead of the assets directory so it can be accessed as a file, but from the point of view of the app, internal storage should be ok.